Kernel/FileSystem/Ext2FileSystem.cpp

```diff
@@ -370,7 +370,7 @@ KResult Ext2FS::write_block_list_for_inode(InodeIndex inode_index, ext2_inode& e
 
             auto ind_block_contents = ByteBuffer::create_uninitialized(block_size());
             if (ind_block_new) {
-                memset(ind_block_contents.data(), 0, dind_block_contents.size());
```

Comment:
Note: Here is one such bug this addresses, we are passing the wrong data size.

Reply:
Great catch!

Reply:
Typo in commit message here btw, s/BlockBuffer/ByteBuffer/